gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -25,7 +25,10 @@ require("./UserLogonHandlers.js");
 require("./BrowserChannel.js");
 require("./DefaultSettingsLoader.js");
 require("./PSPChannel.js");
-require("./CloudBasedFlowManager.js"); // TODO: move this include to be operated by a config
+require("./PrefsServerDataSource.js");
+require("./CloudBasedFlowManager.js");
```

Comment:
See the TODO which was removed - I think the time has come to remove all these includes from here and stick them in a config since none of them are actually referenced by any of the definitions in this file. At the least they should be moved to CloudBasedFlowManager.js but preferably to gpii.flowManager.config.cloud.base.json5 itself

Reply:
I ended up moving the inclusion of CloudBasedFlowManager.js into [flowManager/index.js](https://github.com/GPII/universal/pull/699/files#diff-c7b1202c67ba2815ea05857b71e0e935R10) and moving all cloud related scripts into CloudBasedFlowManager.js. This makes it easier for both integration tests and configs to load scripts for running cloud. For example, `fluid.require("%gpii-universal");` used in integration tests will load all flow manager source code. If CloudBasedFlowManager.js is only moved into the cloud config files, those integration tests will need to include it separately.